What-to-do-if-you-speak-a-language-that-Free-Code-Camp-does-not-yet-support.md

```diff
@@ -4,22 +4,33 @@ We won't be able to add new languages to Free Code Camp until all of our
 challenges are translated into that language.
 
 In addition to translating these initially, we'll also need to maintain 
-the translation as the challenges are gradually updated.
+the translation as the challenges are gradually updated (we use the 
+github issue system to keep record).  
 
 If you're able to help us, please join the gitter channel 
 https://gitter.im/FreeCodeCamp/Translators 
 And also join the channel for translators of your specific language 
 (if it exists):
 
-* Brazilian Portuguese: https://gitter.im/FreeCodeCamp/BrazilianPortuguese
-* Korean: https://gitter.im/StephenMayeux/koreantranslations
-* Spanish: https://gitter.im/AlexTrapp/FreeCodeCampEspa%C3%B1ol
+* Amharic [FreeCodeCamp/TranslationAmharic](https://gitter.im/FreeCodeCamp/TranslationAmharic)
+* Brazilian Portuguese: [FreeCodeCamp/BrazilianPortuguese](https://gitter.im/FreeCodeCamp/BrazilianPortuguese)
+* French [FreeCodeCamp/TranslationFrench](https://gitter.im/FreeCodeCamp/TranslationFrench)
+* German [FreeCodeCamp/TranslationDeutsch](https://gitter.im/FreeCodeCamp/TranslationDeutsch)
+* Hindi [FreeCodeCamp/TranslateHindi](https://gitter.im/FreeCodeCamp/TranslateHindi)
+* Korean [StephenMayeux/koreantranslations](https://gitter.im/StephenMayeux/koreantranslations)
+* Russian [FreeCodeCamp/TranslationRussian](https://gitter.im/FreeCodeCamp/TranslationRussian)
+* Serbian [FreeCodeCamp/TranslateSerbian](https://gitter.im/FreeCodeCamp/TranslateSerbian)
+* Spanish [AlexTrapp/FreeCodeCampEspa%C3%B1ol](https://gitter.im/AlexTrapp/FreeCodeCampEspa%C3%B1ol)
+* Turkish [FreeCodeCamp/TranslateTurkish](https://gitter.im/FreeCodeCamp/TranslateTurkish)
 
-If it doesn't exist we invite you to create one and share it on
-  https://gitter.im/FreeCodeCamp/Translators
+If it doesn't exist please let us know in the 
+[Translators channel](https://gitter.im/FreeCodeCamp/Translators), 
+or create one yourself and share it at the  
+[translators channel](https://gitter.im/FreeCodeCamp/Translators) 
+and at other places to invite more translators and reviewers that speak your 
+language.  
 
-We invite you to read some recommendations based on the spanish team
-experience:
+We invite you to read some recommendations based on the spanish team experience:
   https://github.com/vtamara/fcc_trad/blob/master/Recomendaciones.EN.md

```

Comment:
[Recommendations](https://github.com/vtamara/fcc_trad/blob/master/Recomendaciones.EN.md) or something like that would be nicer.